### ISSUE TITLE
review

### ISSUE THREAD

**Issue by PabloMur** [nicolascasmuz/desafio-modulo-3-nicolascasmuz#1]
Hola! Antes que nada te felicito por llegar a esta instancia! El desafio quedo super bien. Lo único que te diría es que en la parte mobile se rompe un poco, en el sentido de que no se ajusta bien a la pantalla. Después de eso esta joya!

**Comment by Micaela777** [comment]
¡Hola Nicolás! Felicitaciones por llegar hasta acá, tu página está muy bella, sin dudas yo les compraría 10 cactus.
Ahora, en cuanto a consejos que me gustaría añadir, para el futuro te recomiendo que coloques todas esas imágenes en una carpeta, más que nada para que quede más ordenado, y de ahí las vas llamando. Otra cosa, en cada página solo debe haber una sola etiqueta h1, que sería para el título principal, a los demás títulos los podes hacer con un h2.
Referente a lo que menciona PabloMur, cuando abro la pág. en mi celular, el título de la section-2 "Nuestros Productos" no me aparece centrado, sería lo único que veo raro en cuanto a lo visual. Por lo demás está todo muy bien.

**Comment by nicolascasmuz** [comment]
> Hola! Antes que nada te felicito por llegar a esta instancia! El desafio quedo super bien. Lo único que te diría es que en la parte mobile se rompe un poco, en el sentido de que no se ajusta bien a la pantalla. Después de eso esta joya!

Muchas gracias Pablo. Lo de la parte mobile me di cuenta luego de enviar el desafío, cuando lo abrí desde el celu para chequearlo y vi que algunos título se rompían un poco. En la pc con el device toolbar eso no pasaba.

**Comment by nicolascasmuz** [comment]
> ¡Hola Nicolás! Felicitaciones por llegar hasta acá, tu página está muy bella, sin dudas yo les compraría 10 cactus. Ahora, en cuanto a consejos que me gustaría añadir, para el futuro te recomiendo que coloques todas esas imágenes en una carpeta, más que nada para que quede más ordenado, y de ahí las vas llamando. Otra cosa, en cada página solo debe haber una sola etiqueta h1, que sería para el título principal, a los demás títulos los podes hacer con un h2. Referente a lo que menciona PabloMur, cuando abro la pág. en mi celular, el título de la section-2 "Nuestros Productos" no me aparece centrado, sería lo único que veo raro en cuanto a lo visual. Por lo demás está todo muy bien.

Sí, aproveché que tengo el emprendimiento para hacerle el sitio web jaja
Lo del uso de las etiquetas h1, h2... no lo tenía muy claro, pensé que era solo una cuestión de tamaño de los títulos.
Y lo de separar las imágenes en una carpeta a parte me vendrá bien para el futuro.
Muchas gracias por los consejos Micaela.
